WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/ScanViewModelTest.kt

```diff
@@ -126,6 +127,14 @@ class ScanViewModelTest : BaseUnitTest() {
                 assertThat(uiStates.first()).isInstanceOf(FullScreenLoadingUiState::class.java)
             }
 
+    @Test
+    fun `given last scan state present in db, when vm starts, then app displays full screen loading scan state`() =
```

Comment:
Minor (🔍): Since the logic has change and the app always displays the full screen loading scan state when view model starts, I suggest the below:
1) Removing the above test: `given last scan state not present in db, when vm starts, then app displays full screen loading scan state`
2) Renaming this test to: `when vm starts, then app displays full screen loading scan state`

The `given` is no longer necessary as there is no precondition for the `then` to change.

Reply:
Merged in 96566b4